docs/application/native/guides/connectivity/softap.md

```diff
@@ -0,0 +1,342 @@
+# SoftAP
+
+SoftAP means software access point for local area connections without internet access.
+
+The main features of the SoftAP API includes:
+
+- Enable or disable SoftAP
+
+   You can turn SoftAP on or off.
+
+
+- Manage SoftAP
+
+   You can manage the SoftAP configuration settings such as Service Set Identifier (SSID) and passphrase.
+   The following table lists the attributes of SoftAP:
+
+   **Table: The attributes of SoftAP**
+    <table>
+     <thead>
+      <tr>
+       <th>Attributes</th>
+       <th>Description</th>
+      </tr>
+     </thead>
+     <tbody>
+      <tr>
+       <td>SSID</td>
+       <td>Service Set Identifier (SSID)</td>
+      </tr>
+      <tr>
+       <td>Security type</td>
+       <td>None, WPA2_PSK, and WPS</td>
+      </tr>
+      <tr>
+       <td>Passphrase</td>
+       <td>Passphrase for wireless access</td>
+      </tr>
+      <tr>
+       <td>Visibility</td>
+       <td>Visibility of SSID</td>
+      </tr>
+      <tr>
+       <td>IP address</td>
+       <td>Local IP address</td>
+      </tr>
+      <tr>
+       <td>Mode</td>
+       <td>Wireless mode (B, G, A, and AD)</td>
+      </tr>
+      <tr>
+       <td>Channel</td>
+       <td>Channel</td>
+      </tr>
+     </tbody>
+    </table>
+
+- SoftAP client
+
+    You can get information about connected client such as name, IP address, MAC address, and connection time.
+
+
+## Prerequisites
+
+1. To use the SoftAP API (in [mobile](../../api/mobile/latest/group__CAPI_NETWORK_SOFTAP_MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI_NETWORK_SOFTAP_MODULE.html) applications), the application must request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+   ```
+   <privileges>
```

Comment:
Refering https://github.com/Samsung/tizen-docs/blob/master/styleguide/sample1.md, the code also has guide line for the indentation. In case of Code, we use 4 spaces for the indentation but it looks like only 2 spaces are used in the page.

I missed this point at previous review. Would you please change to 4 spaces for the indentation in side of the every code block?